lib/dashboard/DefaultDashboardFactory.ts

```diff
@@ -58,22 +58,30 @@ export interface MonitoringDashboardsProps {
   /**
    * Flag indicating whether the default dashboard should be created.
    * This is independent on other create dashboard flags.
+   *
+   * @default true
    */
-  readonly createDashboard: boolean;
+  readonly createDashboard?: boolean;
   /**
    * Flag indicating whether the summary dashboard should be created.
    * This is independent on other create dashboard flags.
+   *
+   * @default true
    */
   readonly createSummaryDashboard: boolean;
```

Comment:
Should we make these optional as well? It may be nice to have them default to `false` to avoid creating extra resources by default unless the user explicitly wants them.

Reply:
Agree to defaulting them to false except for the `createDashboard` case where in most cases it would be set to true. Still open to it being `false` too for consistency

Reply:
Sorry, it seems I forgot to change the type definitions. They should be optional as well. I was thinking what dashboards should be create by default and I was split between default only and all. Since I am on fence, I am fine with moving towards default only.